Source/Web View/WebViewController.swift

```diff
@@ -292,40 +296,47 @@ extension WebViewController {
      :param: url The URL used to load the web view.
     */
     final public func loadURL(url: NSURL) {
-        webView.stopLoading()
+        if self.dataTask?.state == .Running { // allow running task to finish
```

Comment:
I'm thinking we should assume that if someone calls loadURL() they are intending to cancel whatever current load is happening and load a new URL. I would remove this return and just call `self.dataTask?.cancel()` instead